src/tools/flash.c

```diff
@@ -28,9 +28,9 @@ static void cleanup(int signum) {
 
 static void usage(void)
 {
-    puts("command line:   ./st-flash [--debug] [--reset] [--opt] [--serial <serial>] [--format <format>] [--flash=<fsize>] {read|write} <path> [addr] [size]");
-    puts("command line:   ./st-flash [--debug] [--serial <serial>] erase");
-    puts("command line:   ./st-flash [--debug] [--serial <serial>] reset");
+    puts("command line:   ./st-flash [--debug] [--reset] [--opt] [--serial <serial>] [--format <format>] [--flash=<fsize>] [--freq=<Hz>] {read|write} <path> [addr] [size]");
+    puts("command line:   ./st-flash [--debug] [--freq=<Hz>] [--serial <serial>] erase");
+    puts("command line:   ./st-flash [--debug] [--freq=<Hz>] [--serial <serial>] reset");
```

Comment:
To have the documentation part of the related issue addressed we should also consider an update in our tutorial as well. In my opinion the section "Useful tool options" would be a good place. I wanted to extend that section anyway... There we can also include a table for the available frequency options as well.

Reply:
Indeed, I did some searching, no proper positions were found. Your idea sounds quite good👏

Reply:
Feel free to add such info there at the top. I can do some additional formatting later on, when I continue my work on this tutorial. I may set up a table listing all available cmdline-options which also states for which tool they are available. That would avoid duplicates and provide a clearer appearance.

Reply:
Added, also for `--opt`

Reply:
Very attentive -  Thank you! :+1: